tests/src/Kernel/fixtures/markup_rendering_patterns/badge.yml

```diff
@@ -1,3 +1,28 @@
+badge_with_all_values:
```

Comment:
We have no test coverage for the case where we don't pass settings. Since we don't have default either in the pattern template, this can lead to regressions if BCL changes the values. Let's add a case without any settings.